### DISCUSSION TITLE
Docker sunsetting free team organisations, impact on compas?

### DISCUSSION THREAD

**Discussion by tetov** [compas-dev/compas#1128] [General]
- [FAQ from Docker](https://web.docker.com/rs/790-SSB-375/images/privatereposfaq.pdf)
- [blogpost explainer](https://blog.alexellis.io/docker-is-deleting-open-source-images/)
- [tweet with screenshot of email](https://twitter.com/alexellisuk/status/1635679295891812359?s=20)

Is compas affected? Should we migrate to another image registry?

(Should I post this here or on the forum?)

**Comment by gonzalocasas** [comment]
![image](https://user-images.githubusercontent.com/933277/227656428-4a132f99-8056-4164-ac69-dfc8fd53c7ed.png)

They are rolling back their decision, so, I'm closing this one. It would still be a good idea to consider alternative registries though.
